hw/dv/verilator/cpp/verilator_memutil.cc

```diff
@@ -95,6 +111,8 @@ static void PrintHelp() {
                "-l|--meminit=NAME,FILE[,TYPE]\n"
                "  Initialize memory region NAME with FILE [of TYPE]\n"
                "  TYPE is either 'elf' or 'vmem'\n\n"
+               "--load-elf=FILE\n"
```

Comment:
In order to conform with the current structure, add a short option in
`getopt_long(argc, argv, ":r:m:f:l:E:h", long_options, nullptr);` or maybe with a small letter.

Reply:
Maybe we do not want to do that.
I had some problems again with using short arguments multiple times, same problem happens with the long arguments without '='. I think the first `getopt_long()` invocation reorders `argv` which invalidates the options for an argument. Keeping it space free will keep the parts together and valid. I can do a follow up PR where we can discuss if we want to keep short arguments or remove then were necessary.

Reply:
Mid-air collision! Yep, sounds like a good idea. I've just changed things to match the other arguments, so at least it's uniform. I'm happy to lose the short forms here too.

Reply:
I think removing all short options would be helpful because we silently ignore errors from unknown short options due to the fact how `VerilatorSimCtrl::ParseCommandArgs` forwards the arguments to different parsers.
But I think it makes sense to do that in a separate PR in order to raise some more awareness for the change.
So for now you can either leave `-E` in here or it that again. Sorry for the confusion.